src/util/gobConfig.ml

```diff
@@ -351,7 +351,15 @@ struct
 
   (** Merge configurations form a file with current. *)
   let merge_file fn =
-    let v = Yojson.Safe.from_channel % BatIO.to_input_channel |> File.with_file_in (Fpath.to_string fn) in
+    let file = Fpath.to_string fn in
+    let path = if Sys.file_exists file then file else
+        let configs = List.map Fpath.v Goblint_sites.conf in
+        let r = List.find_opt (fun v -> Fpath.filename fn = Fpath.filename v) configs in
```

Comment:
Matching them by only looking at filename parts of paths is quite a weird way of finding files.
The normal way would be to just look `fn` up like this:
https://github.com/goblint/analyzer/blob/4f57a8dadf8e6b4b9579f5116273acc3d8cf4c04/src/maingoblint.ml#L188-L192
https://github.com/goblint/analyzer/blob/4f57a8dadf8e6b4b9579f5116273acc3d8cf4c04/src/maingoblint.ml#L203-L211